### ISSUE TITLE
diagram

### ISSUE THREAD

**Issue by cbolson1** [tommybananas/mufa-info-api#2]
the app should have a link to the satellite view of the field (that shows the field numbering).  I'm not sure if that feature belongs in this source or a different location

**Comment by tommybananas** [comment]
I agree, this has been on the radar for years. We should either create a
URL scheme based off of field ID, api.com/fields/:Id, or just return an
absolute URL with location object.

We'd also have to update the app to support this, which I'm afraid to do.
Would you like access to the source? It's build with the ionic framework.
Ideally it would be rewritten in v2+ but I haven't had time.
